[BUG] integration_test fails on default check-extension matrix (matrix.services undefined on v5.1.0, template validation error on v6.0.0)

# :bug: Bug report

## Current Behavior

The `integration_test` job in the `check-extension.yaml` reusable workflow fails immediately (before any setup) on both `v5.1.0` and `v6.0.0` when callers use the default `supported-version` output as the `matrix` input.

### `v5.1.0` failure

The reusable workflow references `matrix.services` at line ~202, but the matrix entries emitted by `supported-version@v5.1.0` with `include_services: false` (the default) have no `services` key. Job fails in ~2s with the variable unset.

### `v6.0.0` failure (regression)

v6.0.0 tightened the template so that the now-undefined `matrix.services` expression produces a template-validation error rather than a runtime undefined-variable. The job doesn't even start:

\`\`\`
##[error]The template is not valid. graycoreio/github-actions-magento2/.github/workflows/check-extension.yaml@v6.0.0
(Line: 202, Col: 17): Unexpected value ''
\`\`\`

Neither the v5.1.0 nor the v6.0.0 release notes suggest `include_services: true` is a prerequisite for `integration_test`, and the minimal example in `check-extension-README.md` doesn't mention it either.

## Expected Behavior

Either:

1. The `integration_test` job should degrade gracefully when `services` isn't in the matrix — e.g. skip service setup and run tests with whatever services are preconfigured in the runner, or
2. The `supported-version` action should emit `services: []` (empty list) by default so the template expression resolves cleanly, or
3. Document explicitly in `check-extension-README.md` that `include_services: true` on `supported-version` is a hard prerequisite for consuming `integration_test`.

Related: #209, which reported this in a different shape. It was closed as fixed in 5d47cda, but the fix doesn't cover the "matrix output with `include_services: false` being passed directly into check-extension" case.

## Steps to Reproduce

Call `supported-version@v5.1.0` with no extra inputs (or with `project: mage-os`), then pass the result to `check-extension.yaml@v5.1.0`:

\`\`\`yaml
jobs:
  compute-matrix:
    runs-on: ubuntu-latest
    outputs:
      matrix: \${{ steps.supported.outputs.matrix }}
    steps:
      - uses: actions/checkout@v4
      - uses: graycoreio/github-actions-magento2/supported-version@v5.1.0
        id: supported
        with:
          project: mage-os

  check-extension:
    needs: compute-matrix
    uses: graycoreio/github-actions-magento2/.github/workflows/check-extension.yaml@v5.1.0
    with:
      matrix: \${{ needs.compute-matrix.outputs.matrix }}
\`\`\`

Live reproduction: https://github.com/mage-os-lab/module-ai-base/actions/runs/24708855101/job/72268348772

## Your Environment

- Workflow versions tested: `v5.1.0` (silent failure at runtime), `v6.0.0` (template-validation failure at compile-time)
- `supported-version` defaults used: no `include_services` override

--- a/supported-version/action.yml
+++ b/supported-version/action.yml
@@ -24,7 +24,7 @@ inputs:
 
   include_services:
     required: false
-    default: "false"
+    default: "true"
     description: "Whether to include a `services` key in each matrix entry with GitHub Actions service configurations."
 
 outputs:

